src/main/scala/edu/ie3/powerFactory2psdm/model/powerfactory/PowerFactoryGridMaps.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * © 2021. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.powerFactory2psdm.model.powerfactory
+
+import edu.ie3.powerFactory2psdm.exception.pf.{
+  MissingGridElementException,
+  MissingParameterException
+}
+import edu.ie3.powerFactory2psdm.model.powerfactory.PowerFactoryGrid.{
+  Lines,
+  Nodes,
+  Switches
+}
+import java.util.UUID
+import com.typesafe.scalalogging.LazyLogging
+
+class PowerFactoryGridMaps(pfGrid: PowerFactoryGrid) extends LazyLogging {
+
+  val UUID2node: Map[UUID, Nodes] = pfGrid.nodes match {
```

Comment:
Variable names with upper case latters indicate a static member of a class (in Java). For Scala, it doesn't make a difference, but you should avoid doing this. Moreover, please use `Uuid`, if you do camel-case.

Reply:
Was not sure about the naming part as in the java UUID package they always went for the `randomUUID` type approach.